aws/corehandlers/handlers.go

```diff
@@ -106,6 +106,22 @@ var SendHandler = request.NamedHandler{
 			sender = sendWithoutFollowRedirects
 		}
 
+		if request.NoBody == r.HTTPRequest.Body {
```

Comment:
Should this exist in a handler or in the `request.Send` method? Since users can clear the handler list, possibly moving this to `request.Send` may be a better location? Thoughts?